Godeps/Godeps.json

```diff
@@ -2585,6 +2580,10 @@
 			"ImportPath": "github.com/mxk/go-flowrate/flowrate",
 			"Rev": "cca7078d478f8520f85629ad7c68962d31ed7682"
 		},
+		{
+			"ImportPath": "github.com/nikhita/systemstat",
```

Comment:
Happy to move into another location, just adding this here to get started with this commit.

https://github.com/nikhita/systemstat